packages/astro/src/runtime/server/render/server-islands.ts

```diff
@@ -47,7 +47,7 @@ export function renderServerIsland(
 				}
 			}
 
-			destination.write('<!--server-island-start-->');
```

Comment:
`astro` here isn't anything special... it just needs to be a condition that evaluates to false. I wasn't sure about putting `server-island-start` in the condition part, because I was worried the dashes might break the regex minification libraries rely on to identify these conditional comments. As a bonus, it identifies "hey, Astro put this weird comment here" to anyone who might happen to see it and wonder. But this could be `if false` or `if serverislandstart` just as easily.

Reply:
Fine with me!